public/app/common/components/cc-form/cc-form.jsx

```diff
@@ -0,0 +1,36 @@
+var React = require('react');
+var styleMixin = require('mixins/style-mixin');
+var t = require('tcomb-form');
+var Form = t.form.Form;
+
+var options = {
+  templates: {
+    textbox: require('./ccTextInput.jsx')
+  }
+};
+
+var CCForm = React.createClass({
```

Comment:
As discussed yesterday, we shouldn't use a "CC" prefix if the code is reusable and not only related to the controlcenter